background/style-via-xhr.js

```diff
@@ -0,0 +1,85 @@
+/* global API CHROME prefs */
+'use strict';
+
+// eslint-disable-next-line no-unused-expressions
+CHROME && (async () => {
+  const prefId = 'styleViaXhr';
+  const blobUrlPrefix = 'blob:' + chrome.runtime.getURL('/');
+  const stylesToPass = {};
+
+  await prefs.initializing;
+  toggle(prefId, prefs.get(prefId));
+  prefs.subscribe([prefId], toggle);
+
+  function toggle(key, value) {
+    if (!chrome.declarativeContent) { // not yet granted in options page
+      value = false;
+    }
```

Comment:
It seems that we disable the entire `styleViaXHR` feature when `declarativeContent` is disabled. I thought `declarativeContent` only makes content script runs faster. Won't `styleViaXHR` work without `declarativeContent`?

Reply:
It will do it later when all the other `document_start` scripts run from other extensions and when Chrome parses the HTML.

I didn't want to add a permanent permission that is only used when enabled in options by the user, but I guess it's fine as it doesn't add any warnings so it won't disable the extension on update. Do we need to add a justification for the new permission in the dashboard, @narcolepticinsomniac?